traefik.sample.toml

```diff
@@ -660,17 +660,46 @@
 
 # Kubernetes server endpoint
 #
-# When deployed as a replication controller in Kubernetes,
-# Traefik will use env variable KUBERNETES_SERVICE_HOST
-# and KUBERNETES_SERVICE_PORT_HTTPS as endpoint
+# When deployed as a replication controller in Kubernetes, Traefik will use
+# the environment variables KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT
```

Comment:
Note that I changed `KUBERNETES_SERVICE_PORT_HTTPS` to `KUBERNETES_SERVICE_PORT` because [that's what client-go uses](https://github.com/kubernetes/client-go/blob/release-1.5/1.5/rest/config.go#L257) for its check.